src/shared/utils/index.ts

```diff
@@ -204,3 +204,51 @@ export function blacklistKeys(raw: { [key: string]: any }, keys: string[]) {
       return obj;
     }, {});
 }
+
+/**
+ * True if given date is today, false otherwise
+ */
+export function isToday(date: string): boolean {
+  const today = new Date(Date.now());
+  const givenDate = new Date(date);
+
+  if (givenDate.toString() === 'Invalid Date') {
+    return false;
+  }
+  return (
+    givenDate.getDay() === today.getDay() &&
```

Comment:
This may return incorrect results if Date.now() and the input date have different timezones, better use the UTC versions: `getUTCDay(), getUTCMonth(), getUTCFullYear()` OR make sure that both dates are converted to UTC earlier in the function